src/main/java/step3/racingcarWinner/domain/Car.java

```diff
@@ -0,0 +1,61 @@
+package step3.racingcarWinner.domain;
+
+import java.util.Objects;
+
+public class Car {
+
+    private String name;
+    private int position;
+    private boolean finalWinner = false;
+
+    public Car(String name, int position) {
+        this.name = name;
+        this.position = position;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public boolean isFinalWinner() {
+        return finalWinner;
+    }
+
+    public void move() {
+        this.position++;
+    }
+
+    public void prizeWinner() {
+        this.finalWinner = true;
+    }
+
+    public boolean isEqualPosition(int inputPosition){
+        if(this.position != inputPosition){
```

Comment:
아래의 코드처럼 한 줄로 줄일 수 있어요.

```suggestion
        return this.position != inputPosition
```